app/vmui/packages/vmui/src/utils/query-string.ts

```diff
@@ -10,10 +10,11 @@ export const getQueryStringValue = (
   return values?.[key] || defaultValue || "";
 };
 
-export const getQueryArray = (): string[] => {
+export const getQueryArray = (name: string): string[] => {
```

Comment:
Prefer explicitly narrowing `name` to the allowed values for better type safety.
```suggestion
export const getQueryArray = (name: "expr" | "hide"): string[] => {
```